decidim-core/app/cells/decidim/redesigned_profile/show.erb

```diff
@@ -0,0 +1,29 @@
+<div class="profile__scaffold__top">
+  <div class="container">
+    <div class="profile__scaffold__top-content">
+      <div>
+        <%= render :avatar %>
+      </div>
+      <div>
+        <%= render :details %>
+      </div>
+      <div>
+        <%= cell "decidim/profile_actions", profile_holder, context %>
```

Comment:
The profile actions break for the group profiles when signed in and on mobile:
![Group profile actions on mobile](https://user-images.githubusercontent.com/864340/183035670-dcb384a8-088b-4bfc-827a-109a0be45f7c.png)